planet/__version__.py

```diff
@@ -1 +1,3 @@
-__version__ = '2.11.0dev'
+from importlib.metadata import version
+
+__version__ = version("planet")
```

Comment:
Slick!  I haven't seen this before.  From the docs it looks like `version()` will pull the version number of the package as a string from the pypi distribution package?  Does that mean we don't have to bump the version number manually anymore and this will just pull whatever version we tag?

Reply:
Yes, this makes releasing much simpler!